x-pack/legacy/plugins/siem/public/pages/case/components/create/index.test.tsx

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { Create } from './';
+import { TestProviders } from '../../../../mock';
+import { getFormMock } from '../__mock__/form';
+import { Router, routeData, mockHistory, mockLocation } from '../__mock__/router';
+
+import { useInsertTimeline } from '../../../../components/timeline/insert_timeline_popover/use_insert_timeline';
+import { usePostCase } from '../../../../containers/case/use_post_case';
+jest.mock('../../../../components/timeline/insert_timeline_popover/use_insert_timeline');
+jest.mock('../../../../containers/case/use_post_case');
+import { useForm } from '../../../../../../../../../src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks';
+import { wait } from '../../../../lib/helpers';
+import { SiemPageName } from '../../../home/types';
+jest.mock(
+  '../../../../../../../../../src/plugins/es_ui_shared/static/forms/hook_form_lib/hooks/use_form'
+);
+
+export const useFormMock = useForm as jest.Mock;
+
+const useInsertTimelineMock = useInsertTimeline as jest.Mock;
+const usePostCaseMock = usePostCase as jest.Mock;
+
+const postCase = jest.fn();
+const handleCursorChange = jest.fn();
+const handleOnTimelineChange = jest.fn();
+
+const defaultInsertTimeline = {
+  cursorPosition: {
+    start: 0,
+    end: 0,
+  },
+  handleCursorChange,
+  handleOnTimelineChange,
+};
+const sampleData = {
+  description: 'what a great description',
+  tags: ['coke', 'pepsi'],
+  title: 'what a cool title',
+};
+const defaultPostCase = {
+  isLoading: false,
+  isError: false,
+  caseData: null,
+  postCase,
+};
+describe('Create case', () => {
+  const formHookMock = getFormMock(sampleData);
+
+  beforeEach(() => {
+    jest.resetAllMocks();
+    useInsertTimelineMock.mockImplementation(() => defaultInsertTimeline);
+    usePostCaseMock.mockImplementation(() => defaultPostCase);
+    useFormMock.mockImplementation(() => ({ form: formHookMock }));
+    jest.spyOn(routeData, 'useLocation').mockReturnValue(mockLocation);
+  });
+
```

Comment:
Initial render test :)

Reply:
`.find(`[data-test-subj="create-case-submit"]`)` would not exist if the initial render had an issue. running `.find(`[data-test-subj="create-case-submit"]`)` confirms the initial render